Makefile

```
@@ -60,7 +64,7 @@ clean:
 	-rm build/_output/bin/*
 	-rm coverage*.out
 	-rm report*.json
-	-rm kubeconfig_managed
+	-rm kubeconfig_managed*
```

Looks like this was wrong before this PR since it's a hub cluster, but let's just go ahead and wipe all kubeconfigs out.

```suggestion
	-rm kubeconfig_*
```